morgoth/base/utils.py

```diff
@@ -0,0 +1,70 @@
+import requests
+
+from datetime import datetime
+
+from django.conf import settings
+
+
+class BalrogAPI(object):
+    def __init__(self, auth=None):
+        """ Creates an object which wraps the Balrog REST API."""
+        self.auth = auth
+        self.session = requests.session()
+        self.csrf_token = None
+
+    def _is_csrf_token_expired(self):
+        if not self.csrf_token:
+            return True
+
+        expiry = self.csrf_token.split('##')[0]
+        if expiry <= datetime.now().strftime('%Y%m%d%H%M%S'):
+            return True
+
+        return False
+
+    def _make_request(self, url, data, method):
+        headers = {'Accept-Encoding': 'application/json',
+                   'Accept': 'application/json'}
+
+        url = '%s%s' % (settings.BALROG_API_BASE_URL, url)
+
+        req = self.session.request(method=method, url=url, data=data,
+                                   timeout=settings.BALROG_API_REQUEST_TIMEOUT,
+                                   verify=False, auth=self.auth, headers=headers)
```

Comment:
`verify=False` is probably a bad thing to have hard coded. I assume this is for talking to a local balrog api? This should probably be controlled by a setting.